index.d.ts

```diff
@@ -46,7 +46,7 @@ declare module 'react-native-toast-message' {
 
   export default class Toast extends React.Component<ToastProps> {
     static show(options: {
-      type: string,
+      type: 'success | error | info',
```

Comment:
I think we should add `string` as well, to support custom types that might get created via the `config` prop

```suggestion
      type: 'success' | 'error' | 'info' | string,
```